infrastructure/hosts/catcolab-next/backend.nix

```diff
@@ -183,17 +210,28 @@ in {
     };
 
     security.sudo.extraRules = [{
-        users = [ "catcolab" ]; 
+        users = [ "catcolab" "deployuser" ];
         commands = [
             { command = "/run/current-system/sw/bin/systemctl start automerge"; options = [ "NOPASSWD" ]; } 
             { command = "/run/current-system/sw/bin/systemctl stop automerge"; options = [ "NOPASSWD" ]; } 
             { command = "/run/current-system/sw/bin/systemctl restart automerge"; options = [ "NOPASSWD" ]; }
             { command = "/run/current-system/sw/bin/systemctl start backend"; options = [ "NOPASSWD" ]; } 
             { command = "/run/current-system/sw/bin/systemctl stop backend"; options = [ "NOPASSWD" ]; } 
             { command = "/run/current-system/sw/bin/systemctl restart backend"; options = [ "NOPASSWD" ]; }
+            { command = "${updateScript}/bin/catcolab-update";  options = [ "NOPASSWD" ]; }
         ]; 
     }];
 
+
+    users.users.deployuser = {
+        isNormalUser = true;
+        openssh.authorizedKeys.keys = [
+            ''
+                command="${updateScript}/bin/catcolab-update",no-port-forwarding,no-agent-forwarding,no-X11-forwarding,no-pty ${catcolabnextDeployuserKey}
```

Comment:
Declaration and use of the existing scripts is strange blend of manual unixy install steps and declarative nixy configuration. Mixing them seems wrong so I added the new one the nix way. I'm not sure if this was the right call.